src/js/common/components/form-elements/ErrorableCurrentOrPastDate.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import moment from 'moment';
+
+import ErrorableDate from './ErrorableDate';
+import { isDirtyDate } from '../../utils/validations';
+import { dateToMoment } from '../../utils/helpers';
+
+export default function ErrorableCurrentOrPastDate(props) {
```

Comment:
These are the props that are passed to `ErrorableCurrentOrPastDate. If you write React components using their function syntax, you get the props passed as a parameter, instead of using `this.props`.